fetch-jars

```diff
@@ -3,26 +3,62 @@
 set -eu
 
 # Usage: ./fetch-jars [rev-name] [--copper]
+# If a revision is not specified, the script fetches the latest jars from the current branch,
+# falling back to develop.
+# If --copper is specified, only fetch the Copper jars.
 
+COMMIT_ARTIFACTS="https://foundry.remexre.xyz/commit-artifacts/"
 
+function has_jars {
+  wget --spider -q "$COMMIT_ARTIFACTS/$1"
+}
+
+rev=""
+
+# Look for jars in the specified revision
 if [[ $* != *--copper* && $# -gt 0 || $# -gt 1 ]]; then
-    rev=$(git rev-parse $1)
-    echo "Warning: Fetching unstable jars!  (commit $rev)"
+  rev=$(git rev-parse $1)
+fi
+
+# Get the hash of the latest commit on develop.
+# This always gives the latest commit regardless of if the repo is up to date,
+# and works in a Jenkins checkout.
+DEVELOP=$(git ls-remote https://github.com/melt-umn/silver develop | cut -f1)
+
+# Look for jars in the current commit and its parents.
+# First, check that we are inside a git repo.
+if [[ -z $rev ]] && git rev-parse --is-inside-work-tree 1> /dev/null 2> /dev/null; then
+  for commit in $(git rev-list HEAD ^$DEVELOP); do
+    if has_jars $commit; then
+      echo "Found jars from past commit"
```

Comment:
Maybe log which one it is, at least if it's not HEAD

Reply:
That gets printed with git show on line 46